airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java

```diff
@@ -292,6 +292,16 @@ public void testEmptyStateIncrementalIdenticalToFullRefresh() throws Exception {
     assertSameRecords(fullRefreshRecords, emptyStateRecords, assertionMessage);
   }
 
+  /**
+   * In order to launch a source on Kubernetes in a pod, we need to be able to wrap the entrypoint.
+   * The source connector must specify its entrypoint in the AIRBYTE_ENTRYPOINT variable. This test
+   * ensures that the entrypoint environment variable is set.
+   */
+  @Test
+  public void testEntrypointEnvVar() throws Exception {
```

Comment:
this should also live in the [python source acceptance test suite](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py)

Do you feel comfortable making that change? Alternatively can offload to connectors teams

Reply:
I'd prefer to offload to connector teams. Should be a lot faster for them to implement.

https://github.com/airbytehq/airbyte/issues/3903 is the main issue. Can you create another ticket if necessary?